engines/director/lingo/lingo-object.cpp

```diff
@@ -522,6 +522,10 @@ Datum Window::getField(int field) {
 		return getWindowType();
 	case kTheRect:
 		return getStageRect();
+	case kTheModal:
+		return getModal();
+	case kTheFileName:
+		break;
```

Comment:
This introduces a case where the function can return without returning a value, which clang flags as a warning:

```
    C++      engines/director/lingo/lingo-object.o
engines/director/lingo/lingo-object.cpp:534:1: warning: non-void function does not return a value in all control paths [-Wreturn-type]
}
^
```